content/en/docs/reference/kubectl/resource_printing/cluster_information.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Cluster Information"
+linkTitle: "Cluster Information"
+weight: 7
+type: docs
+description: >
+    Prints Cluster Information
+---
+
+
+{{< alert color="success" title="TL;DR" >}}
+- Print information about the Cluster and Client versions
+- Print information about the Control Plane
+- Print information about Nodes
+- Print information about APIs
+{{< /alert >}}
+
+# Cluster Info
```

Comment:
Thanks.
We've realized that a straight conversion isn't right. This proposed tweak should help a lot though:
```suggestion
```

Reply:
Done!